lxd/db/storage_volumes.go

```diff
@@ -171,6 +171,7 @@ type StorageVolumeFilter struct {
 	Project *string
 	Name    *string
 	PoolID  *int64
+	UUIDs   *[]string
```

Comment:
UUIDs doesnt need to be a pointer as you can have a nil slice and thus it can be made optional without having a pointer.